angular2/src/angular2TextMask.ts

```diff
@@ -45,11 +45,11 @@ export class MaskedInputDirective implements ControlValueAccessor {
       this.inputElement = this.element.nativeElement.getElementsByTagName('INPUT')[0]
     }
 
-    if (this.inputElement) {
-      this.textMaskInputElement = createTextMaskInputElement(
-          Object.assign({inputElement: this.inputElement}, this.textMaskConfig)
-      )
-    }
+    this.textMaskInputElement = createTextMaskInputElement({})
```

Comment:
`?` is the optional modifier.

`export default function createTextMaskInputElement(a?: any): any`